File: E02/C++/E02.2T01.cpp
///////////////////////////////////
// Lab 1. 	Choose a Drink
///////////////////////////////////

#include <iostream>
#include <string>
using namespace std;

int main() {

	string professions[] = { "Athlete", "Bisnessman", "Bisnesswoman", "SoftUni Student" };
	string drings[] = { "Wother", "Coffe", "Coffe", "Beer", "Tea" };

	string profession;
	cout << "Enter profession: ";
	getline(cin, profession);

	bool isSet = false;

	for (int index = 0; index < 4; index++) {
		if (!profession.compare(professions[index])) {
			cout << drings[index] << endl;
			isSet = true;
			break;
		}
	}

	if (!isSet) {
		cout << drings[4] << endl;
	}

	system("pause");
	return 0;
}

File: E02/Java/E02.2T01.java
/////////////////////////////////////////////////
// Choose A Drink
/////////////////////////////////////////////////

import java.util.Scanner;

public class ChooseADrink {

    public static void main(String[] args) {
		
        Scanner scanner = new Scanner(System.in);
        String profession = scanner.nextLine();
        String result = "";
        switch(profession){
            case "Athlete": result = "Wather";
            break;
            case "Bisnessman": 
            case "Bisnesswoman":  result = "Coffe";
            break;
            case "SoftUni Student": result = "Beer";
            break;
            default: result = "Tea";
            break;
        }
        System.out.println(result);
    }
}